src/main/java/io/projectdiscovery/nuclei/gui/TemplateGeneratorTab.java

```diff
@@ -121,14 +126,14 @@ private String createCommand(Path nucleiPath, URL targetUrl) {
         }
 
         return String.format("%s -v -t %s -u %s",
-                             wrapWithQuotesIfNecessary(nucleiPath.toString()),
-                             wrapWithQuotesIfNecessary(this.templatePath.toString()),
-                             wrapWithQuotesIfNecessary(targetUrl.toString()));
+                wrapWithQuotesIfNecessary(nucleiPath.toString()),
+                wrapWithQuotesIfNecessary(this.templatePath.toString()),
+                wrapWithQuotesIfNecessary(targetUrl.toString()));
     }
 
     private static String wrapWithQuotesIfNecessary(String input) {
         return input.contains(" ") ? String.format("\"%s\"", input)
-                                   : input;
+                : input;
```

Comment:
The original formatting was more readable...You should consider reverting the changes that do not relate to the functionality you have implemented.